.travis/osx..script.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
         # Workaround; No FindQt5.cmake module exists
-        export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        export CMAKE_PREFIX_PATH
 fi
 
-cmake $CMAKE_FLAGS -DUSE_WERROR=OFF ..
+cmake "$CMAKE_FLAGS" -DUSE_WERROR=OFF ..
```

Comment:
Please don't rely on Travis-CI, but test on your own.  If there's one or none parameters, you will get a false positive.